Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -28,7 +29,7 @@ public ShooterWheelSubsystem(
             WPIFactory factory,
             XPropertyManager propManager) {
         super(
-                side+"ShooterWheel",
+                side + " ShooterWheel",
```

Comment:
If we're going to add one space, we might as well put spaces between every word.